include/RAJA/internal/IndexSet.hxx

```diff
@@ -55,8 +55,8 @@
 
 #include "RAJA/config.hxx"
 
-#include "RAJA/IndexSetSegInfo.hxx"
-#include "RAJA/RAJAVec.hxx"
+#include "RAJA/internal/IndexSetSegInfo.hxx"
```

Comment:
It doesn't make sense to me to have IndexSet and Segment headers in internal. Is this another case of putting everything in there and selectively pulling things back out?